pkg/controllers/clusterresourceplacementeviction/controller.go

```diff
@@ -277,7 +282,7 @@ func isEvictionAllowed(bindings []placementv1beta1.ClusterResourceBinding, crp p
 		desiredBindings = int(*crp.Spec.Policy.NumberOfClusters)
 	case placementv1beta1.PickFixedPlacementType:
```

Comment:
Why do we allow eviction on fixed type? It will be put back

Reply:
Will handle this case in a follow up PR